DigitalLearningSolutions.Web/Controllers/LearningPortalController/Current.cs

```diff
@@ -168,6 +170,12 @@ public IActionResult RequestUnlock(int progressId)
         public async Task<IActionResult> MarkActionPlanResourceAsComplete(int learningLogItemId)
         {
             var actionPlanResource = await actionPlanService.GetActionPlanResource(learningLogItemId);
+
+            if (actionPlanResource == null)

```

Comment:
Nice, are these NotFound cases mentioned on the ticket as stuff which should be tested?